asyncpg/connect_utils.py

```diff
@@ -208,6 +209,16 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
                 if passfile is None:
                     passfile = val
 
+            if 'sslmode' in query:
+                val = query.pop('sslmode')
+                if ssl is None:
+                    ssl = val
+
+            if 'ssl' in query:
```

Comment:
It doesn't seem useful to support the boolean `ssl` alongside the `sslmode` enumeration in the DSN.  Let's stick to `sslmode` and other `ssl*` strings.  `ssl=true` is equivalent to `sslmode=verify-full`